src/mumble/MainWindow.h

```diff
@@ -314,6 +314,7 @@ public slots:
 	void trayAboutToShow();
 	void sendChatbarMessage(QString msg);
 	void sendChatbarText(QString msg);
+	void sendChatbarPlainText(QString msg);
```

Comment:
Imo a better approach would be to add a parameter (maybe optional) to `sendChatbarText` to trigger plain-text sending:
```cpp
void sendChatbarText(QString msg, bool plainText = false);
```
and then check for that parameter inside the implementation to see if one goes through Markdown processing or instead through the plain-text processing.

Reply:
Good suggestion, implemented. Had to modify the signal signature, too, as slots cannot have more arguments than signals.

Reply:
Actually, given that you are using the `SIGNAL` and `SLOT` macros to connect them, I think a slot with default arguments should be supported. That's only not supported when using the `connect` syntax working on function pointers instead of macros.
Could you try if you get it to compile without adding a dummy argument to the signal?

Reply:
Using my latest commit as the starting point, with solely changing the connect line to this:
```c++
connect(qteChat, SIGNAL(ctrlEnterPressed(QString, bool)), this, SLOT(sendChatbarText(QString)));
```
`plainText` is always false inside `sendChatbarText` even when the `ctrlEnterPressed` signal is emitted.

Reply:
Ah right - we don't want the default parameter after all :facepalm: 
In that case, I think we can use a lambda like this:
```cpp
connect(qteChat, &ChatbarTextEdit::ctrlEnterPressed, [this](const QString &msg) { sendChatbarText(msg, true); });
```

Reply:
Ah right, Qt5 has new overloads for `QObject::connect()`. That seems to have done the trick.